app/views/estimate_flow/tribunal.html.slim

```diff
@@ -0,0 +1,18 @@
+- content_for :page_title
+  = t(".title")
+- content_for :back do
+  = back_link(:tribunal, @estimate, @back_buttons_invoke_browser_back_behaviour)
+
+.govuk-grid-column-two-thirds
+  = form_for(@form, url: wizard_path, method: :put) do |form|
+    = form.govuk_error_summary t("generic.error_summary_title")
+    = form.govuk_radio_buttons_fieldset :upper_tribunal,
+                                        legend: { text: t(".title"), size: "m", tag: "h1" }
```

Comment:
as this is the main text in the screen should the size be "l"?

Reply:
I had the same thought, but apparently it's a deliberate choice - the text is so long that they'd rather make the text smaller on this one screen: https://dsdmoj.atlassian.net/browse/EL-694?focusedCommentId=267762